### PR TITLE
[5.4] Normalize PhpRedis GET/MGET call

### PR DESCRIPTION
**Pull request by tillkruss**
Fixes https://github.com/laravel/framework/issues/17188.

### CODE REVIEW

File: src/Illuminate/Redis/Connections/PhpRedisConnection.php [hunk 1]
```diff
@@ -18,6 +18,19 @@ public function __construct($client)
     }
 
     /**
+     * Returns the value of the given key.
+     *
+     * @param  string $key
```
Comment (GrahamCampbell) [inline src/Illuminate/Redis/Connections/PhpRedisConnection.php]: cs

**Review by GrahamCampbell — DISMISSED** [review]

File: src/Illuminate/Redis/Connections/PhpRedisConnection.php (hunk 1 above)
Comment (tillkruss) [inline src/Illuminate/Redis/Connections/PhpRedisConnection.php]: Fixed.

**Comment by taylorotwell** [comment]
Honestly I think I'll just document that false and null should not be cached. Or, if you need to cache such a value to wrap it in an array data structure or something like ['foo' => null].

**Comment by tillkruss** [comment]
Works for me.

**Comment by GrahamCampbell** [comment]
I think this is probably the best solution out of the PRs we have?

**Pull request merged**